data/json/items/resources/tailoring.json

```diff
@@ -14,5 +14,332 @@
+    "color": "dark_gray",
+    "count": 100,
+    "ammo_type": "components"
+  },
+  {
+    "id": "sheet_lycra_patchwork",
+    "copy-from": "sheet_lycra",
+    "type": "AMMO",
+    "name": "patchwork lycra clothing parts",
+    "name_plural": "patchwork lycra clothing parts",
+    "description": "A selection of various clothing parts, sewn together from lycra patches in a patchwork fashion.  Suitable for making most clothing, though it's much less time-efficient than if using proper material sheets.",
+    "weight": "60 g",
+    "volume": "60 ml",
+    "price": 1000,
+    "count": 1
+  },
+  {
+    "id": "sheet_neoprene",
+    "type": "AMMO",
+    "category": "spare_parts",
+    "name": "neoprene sheet",
+    "description": "A sheet of neoprene, a synthetic rubber, suitable for making underwater gear.",
+    "weight": "6 g",
+    "volume": "300 ml",
+    "price": 5000,
+    "material": "neoprene",
+    "symbol": "=",
+    "color": "dark_gray",
+    "count": 100,
+    "ammo_type": "components"
+  },
+  {
+    "id": "sheet_neoprene_patchwork",
+    "copy-from": "sheet_neoprene",
+    "type": "AMMO",
+    "name": "patchwork neoprene clothing parts",
+    "name_plural": "patchwork neoprene clothing parts",
+    "description": "A selection of various clothing parts, sewn together from neoprene patches in a patchwork fashion, with waterproofed seams.  Suitable for making most clothing, though it's much less time-efficient than if using proper material sheets.",
+    "weight": "120 g",
+    "volume": "60 ml",
+    "price": 1000,
+    "count": 1
+  },
+  {
+    "id": "sheet_nomex",
+    "type": "AMMO",
+    "category": "spare_parts",
+    "name": "Nomex sheet",
+    "description": "A sheet of Nomex synthetic fabric, suitable for making heat-resistant clothing.",
+    "weight": "5 g",
+    "volume": "300 ml",
+    "price": 15000,
+    "material": "nomex",
+    "symbol": "=",
+    "color": "dark_gray",
+    "count": 100,
+    "ammo_type": "components"
+  },
+  {
+    "id": "sheet_nomex_patchwork",
+    "copy-from": "sheet_nomex",
+    "type": "AMMO",
+    "name": "patchwork Nomex clothing parts",
+    "name_plural": "patchwork Nomex clothing parts",
+    "description": "A selection of various clothing parts, sewn together with Nomex thread from Nomex patches in a patchwork fashion.  Suitable for making most clothing, though it's much less time-efficient than if using proper material sheets.",
+    "weight": "100 g",
+    "volume": "60 ml",
+    "price": 3000,
+    "count": 1
+  },
+  {
+    "id": "sheet_nylon",
+    "type": "AMMO",
+    "category": "spare_parts",
+    "name": "synthetic fabric sheet",
+    "description": "A sheet of synthetic fabric, suitable for making clothing.",
+    "weight": "3 g",
+    "volume": "300 ml",
+    "price": 5000,
+    "material": "nylon",
+    "symbol": "=",
+    "color": "dark_gray",
+    "count": 100,
+    "ammo_type": "components"
+  },
+  {
+    "id": "sheet_nomex_patchwork",
+    "copy-from": "sheet_nomex",
```

Comment:
I think you meant the `id` to be `sheet_nylon_patchwork` and the `copy-from` to be `sheet_nylon`.